src/components/AccordionComponent/AccordionComponent.jsx

```diff
@@ -64,23 +73,24 @@ export const AccordionComponent = (props) => {
             size={{ base: 'sm', md: 'md' }}
             w={{ base: 8, md: 10 }}
             top={2}
-            right={4}
+            right={2}
             variant="danger"
           >
             <DeleteIcon />
           </Button>
 
           <AccordionPanel p={4}>
             <HStack>
-              <Text px={2}>{props.content}</Text>
+              <Text px={2}>{props.name}</Text>
+              {/* ----------Subcategories--------------
```

Comment:
explain please why it's commented?

Reply:
There should be subcategories here, I removed everything that is commented